Userland/Libraries/LibWeb/CSS/CSSStyleDeclaration.cpp

```diff
@@ -26,12 +26,23 @@ String CSSStyleDeclaration::item(size_t index) const
     return CSS::string_from_property_id(m_properties[index].property_id);
 }
 
+NonnullRefPtr<ElementInlineCSSStyleDeclaration> CSSStyleDeclaration::to_inline_declaration(DOM::Element& element)
+{
+    return ElementInlineCSSStyleDeclaration::create(element, move(m_properties), move(m_custom_properties));
+}
```

Comment:
This doesn't seem like a generally useful operation (converting a `CSSStyleDeclaration` into an `ElementInlineCSSStyleDeclaration`)
Instead, let's have a way to construct `ElementInlineCSSStyleDeclaration` by passing it a `CSSStyleDeclaration` that it can take the properties from.